packages/bolt-connection/test/bolt/bolt-protocol-v4x0.test.js

```diff
@@ -589,6 +589,15 @@ describe('#unit BoltProtocolV4x0', () => {
       expect(unpacked).toEqual(object)
     })
   })
+
+  describe('Bolt v5.1', () => {
```

Comment:
Do we not need the other message (logon)

Reply:
I only need to check outside the protocol level if driver supports logoff (which means also it support logon). I might be the case I just called `supportReauth`.